core/extension/extension_api_dump.cpp

```diff
@@ -479,6 +499,14 @@ Dictionary GDExtensionAPIDump::generate_extension_api() {
 				d["name"] = name;
 				d["value"] = value;
 				d["is_bitfield"] = bitfield;
+				if (p_include_docs) {
+					for (const DocData::ConstantDoc &constant_doc : global_scope_doc->constants) {
+						if (constant_doc.name == name) {
+							d["documentation"] = fix_doc_description(constant_doc.description);
+							break;
+						}
+					}
+				}
```

Comment:
This runs in O(n<sup>2</sup>) on the number of constants... which I guess is OK? This number is typically low (dozens)?

(Same for other cases)

Reply:
Personally, I think this is probably fine. This is a "batch process" that's running "offline", so it doesn't need to be fast (within reason). I timed running `--dump-extension-api-with-docs` couple of times locally, and it's taking about 3 seconds on average on my system.